### PR TITLE
Add entity.id to argparse for int Conversion

### PR DESCRIPTION
**Pull request by rcsalome**
This is a fix for #106

Args that are not picked up by the argparse are being treated as
strings. A way around this is to add the specific argument,
'--entity.id', to argparse and treat it as an 'int'. We also suppress
the argument when displaying the "help" message.

### CODE REVIEW

**Comment by rcsalome** [comment]
Actually, I just realized that I haven't properly tested this. I'll try to get test data in my account to see if the `linode-cli events list --entity.type linode --entity.id <id_number>` command works.

But it does seem that the original 400 error is gone.

**Comment by rcsalome** [comment]
The latest commit message is more accurate.

```
This is a fix for #106

It creates a list "INT_FILTER_ARGS" where any argument added to it will
have an argparse argument created that will treat it as an "int" instead
of a "string".

**Comment by Dorthu** [comment]
I'll have to test this approach, but ideally the CLI doesn't need hardcoded values to designate type; this is generated off of [the spec](https://developers.linode.com/api/docs/v4/openapi.yaml) which should be the authority on the types of arguments.

**Comment by rcsalome** [comment]
Thanks for the tip. I looked at the spec, and you're right. It has the `type` in there. Specifically, it shows that the `events` `list` action returns `entity` of type `object` and those have an `id` of type `integer`.

Here is a more robust approach that uses that `type` for the argparse arguments.

File: linodecli/cli.py [hunk 1]
```diff
@@ -133,6 +133,7 @@ def _parse_properties(self, node, prefix=[]):
                     '.'.join(prefix+[name]),
                     info.get('x-linode-filterable') or False,
                     info.get('x-linode-cli-display') or False,
+                    info.get('type') or 'string',
```
Comment (rcsalome) [inline linodecli/cli.py]: I think this is what is currently happening in a way. So this line preserves the default of `string` as a fallback.

**Pull request merged**